### DISCUSSION TITLE
Need an official guide to install code-server on Windows

### DISCUSSION THREAD

**Discussion by CaledoniaProject** [coder/code-server#5991] [Q&A]
Does it support windows?
https://www.npmjs.com/package/code-server

**Comment by lg-nineoneninethreenine** [comment]
Well, yes and no? You can use Cygwin/WSL(preferred) for code-server.

**Reply by CaledoniaProject** [reply]
Yes I can use cygwin, is that recommended officially?

**Reply by lg-nineoneninethreenine** [reply]
Well, no, as it doesn't have a package manager.

**Reply by lg-nineoneninethreenine** [reply]
Try WSL, it works.

**Comment by CaledoniaProject** [comment]
Windows is unsupported? I didn't find any release ... https://github.com/coder/code-server/releases/tag/v4.9.1

**Reply by code-asher** [reply]
We have no Windows support at the moment.  You could try installing from npm but I think it will probably not work.  The WSL suggestion is a good one.
